docs/docsite/rst/intro_getting_started.rst

```diff
@@ -107,6 +108,21 @@ explore what you can do with different modules, and to learn about the Ansible
 also has powerful configuration management and deployment features.  There's more to
 explore, but you already have a fully working infrastructure!
 
+Tips
+
+Only the local server, you do not need to write on inventory file to specify the local server.
```

Comment:
To reference the machine on which you are running Ansible (commonly known as the 'controller'), you don't need to specify it in the inventory as Ansible provides an 'implicit localhost'.